components/domains/renewal.tsx

```diff
@@ -182,6 +180,16 @@ const Renewal: FunctionComponent<RenewalProps> = ({ groups }) => {
         calls.unshift(registrationCalls.vatTransfer(salesTaxAmount)); // IMPORTANT: We use unshift to put the call at the beginning of the array
       }
 
+      if (renewalBox) {
+        calls.push(
+          ...registrationCalls.multiCallAutoRenewal(
```

Comment:
Might be better to change that because we don't do the same for the other `registrationCalls`